web/src/message_fetch.ts

```diff
@@ -277,7 +353,6 @@ export function load_messages(opts, attempt = 1) {
         });
     }
 
-    data.client_gravatar = true;
```

Comment:
I removed this because I believe we should add it directly to the object when defining it.

Reply:
Yeah that seems correct.